jet/hazelcast-3-connector/README.md

```diff
@@ -0,0 +1,138 @@
+= Hazelcast 3 Connector Sample
+
+This example shows how to use the Hazelcast 3 connector to transfer data
+between Hazelcast version 3.x and Hazelcast 5.x+ and back, or use 
+Hazelcast version 3.x for enrichment.
+
+The purpose of the Hazelcast 3 connector is to provide a way to evaluate
+Hazelcast 5 features on your existing data present in Hazelcast 3 
+cluster.
+
+In this demo we will do the following:
+- Start Hazelcast 3.x using the `Hz3MemberWithData` class - this member 
```

Comment:
Minor - we are using a couple of ways to refer to Hazelcast - "Hazelcast 3.x", "Hazelcast version 3.x", "Hz 3", "Hazelcast 3 ", It seems it would be better to use a single one